core/src/main/java/com/yahoo/oak/BasicChunk.java

```diff
@@ -21,27 +21,34 @@
 
 
     /*-------------- Members --------------*/
-    // to compare serilized and object keys
+    protected final OakSharedConfig<K, V> config;
+
+    // to compare serialized and object keys
```

Comment:
Each instance holds its own references to the "config items" it needs.
I implemented it that way to avoid another pointer dereference each time we access these elements.
In addition, the config object is stored here to be passed to the child Chunk instance.